documentation/TechnicalDesignDocument.tex

```diff
@@ -209,8 +228,35 @@
     In this section build automation and its constituent components will be discussed.
     How we applied it, the software used and the targets and platforms supported are among a few.
 
+    The requirements for the project contain several important matters.
+    The project needs to be able to be developed and built from all OSes.
+    It needs to be able to switch hands between multiple developers easily, due to it being open source.
+    It should also obviously be thoroughly tested.
+    Because of this, the team, together with Chide decided that a continuous integration solution was necessary, and have been working on setting this up during the first part of the project.
+
+    The chosen CI system is Travis CI .
+    It is free for open source, and easy to use and learn.
```

Comment:
What do you mean? Free for open source licensed projects? Or free and open source?